kotsadm/pkg/handlers/license.go

```diff
@@ -464,7 +468,9 @@ func GetOnlineInstallStatus(w http.ResponseWriter, r *http.Request) {
 	status, err := online.GetInstallStatus()
 	if err != nil {
 		logger.Error(err)
-		w.WriteHeader(500)
+		JSON(w, 500, GetOnlineInstallStatusErrorResponse{
```

Comment:
this needs to be added here as well, with a `401` status code: https://github.com/replicatedhq/kots/pull/902/files#diff-963b65faecc60bcb8286f696a69cf87eR463